### DISCUSSION TITLE
plot_gsva_heatmap default pathway ranking

### DISCUSSION THREAD

**Discussion by YizhenChen** [reactome/ReactomeGSA#70] [Q&A] [answered]
Hi,

What is the default pathway ranking for the plot_gsva_heatmap function if I did not specify the pathway IDs? It is neither the pathway ID number nor the results from the "maximum differently expressed pathway" based on my observation.  Thank you very much!

**Comment by YizhenChen** [comment, marked as answer]
Oops. I found it first selected a number of pathways based on the maximum differently expressed pathway result, then ranked them by the pathway ID. I just plotted too many pathways at once and overlooked some information. I understand now. Please delete the post if it is misleading.

**Comment by jgriss** [comment]
Dear @YizhenChen 

Thanks a lot for posting the solution!
